File: tools/benchmarks/merlo/benchmarks/stage06p_extended/corpus/text_bytes_utf8/go/main.go
package main
import("fmt";"os";"strconv")
func run(n uint64)uint64{text:=[]byte{109,195,169,108,240,159,152,128,10};sum:=uint64(14695981039346656037);for i:=uint64(0);i<n;i++{sum^=uint64(text[i%9]);sum*=1099511628211};return sum}
func main(){n,e:=strconv.ParseUint(os.Args[1],10,64);if e!=nil{panic(e)};fmt.Printf("%d\n",run(n))}

File: tools/benchmarks/merlo/benchmarks/stage06p_extended/corpus/text_bytes_utf8/rust/main.rs
fn run(n:u64)->u64{let text:[u8;9]=[109,195,169,108,240,159,152,128,10];let mut sum=14695981039346656037u64;for i in 0..n{sum^=text[(i%9)as usize]as u64;sum=sum.wrapping_mul(1099511628211);}sum}
fn main(){let n=std::env::args().nth(1).unwrap().parse::<u64>().unwrap();println!("{}",run(n));}
